recipes/giflib/5.2.x/conanfile.py

```diff
@@ -57,11 +59,10 @@ def package(self):
         cmake.install()
 
         self.copy("COPYING", src=self._source_subfolder, dst="licenses")
-    
+
     def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "GIF"
+        self.cpp_info.names["cmake_find_package_multi"] = "GIF"
         self.cpp_info.libs = tools.collect_libs(self)
-
-        if self.options.shared:
-            self.cpp_info.defines.append("USE_GIF_DLL")
-        else:
-            self.cpp_info.defines.append("USE_GIF_LIB")
+        if self.settings.compiler == "Visual Studio":
```

Comment:
Does mingw need this too?
```suggestion
        if self.settings.os == "Windows":
```

Reply:
No